AuthenticatorBridgeKit/Tests/AuthenticatorBridgeItemDataTests.swift

```diff
@@ -15,6 +14,7 @@ final class AuthenticatorBridgeItemDataTests: AuthenticatorBridgeKitTestCase {
     var dataStore: AuthenticatorBridgeDataStore!
     var errorReporter: ErrorReporter!
     var itemService: AuthenticatorBridgeItemService!
+    var sharedTimeoutService: MockSharedTimeoutService!
```

Comment:
⛏️ This needs to be tear down. Moreover it seems there are other properties needing tear down.
However, I think for this specific service you could just add the mock inline as with `MockSharedKeychainRepository` as you're not using the property anywhere else other than the `setUp`.